calculation/sha_calc/sha_calc/directivity/bea20/bench_data/create_bench.py

```diff
@@ -7,7 +7,7 @@
 from sha_calc.directivity.bea20 import directivity
 
 FAULTS = ["AlpineK2T", "Ashley", "Browning", "Hossack"]
-SRF_LOCATION = Path("/mnt/mantle_data/seistech")  # TODO Change to virtual srfs
+SRF_LOCATION = Path("/mnt/mantle_data/seistech")
```

Comment:
Do we still use this path?

Reply:
Technically yes due to the benchmark tests still running on the srf files, however potentially this should change to using the NHM so they are the same calculation as applied on dbs.

If so then potentially some other functions such as `compute_directivity_srf_multi` in directivity.py would be redundant and could be removed? As well as `_srf_pre_process`?

@claudio525 Keen to hear your thoughts on this

Reply:
Yes, change to NHM and get rid of the srf stuff, unless there is a good reason to keep it? But I think we will only be using it with NHM in the future, so no point in keeping it